action.yml

```diff
@@ -207,38 +207,46 @@ runs:
       env:
         IMAGE_NAME: ${{env.AUTO_TAG}}
 
+    - name: Generate Random String to use for tag list json
```

Comment:
Is it easier to use the `GITHUB_RUN_ID` instead? https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Chatted about this in slack, that ID is the same across multiple matrix jobs so it's not unique